module-apps/application-meditation/windows/MeditationOptionsWindows.cpp

```diff
@@ -122,9 +122,9 @@ void PreparationTimeWindow::onBeforeShow([[maybe_unused]] ShowMode mode, [[maybe
 unsigned int PreparationTimeWindow::getSelectedItemIndex() const
 {
     auto app                = static_cast<app::ApplicationMeditation *>(application);
-    const auto selectedTime = app->state->preparationTime;
-    for (unsigned int i = 0; i < preparationTimes.size(); ++i) {
-        if (selectedTime == preparationTimes[i]) {
+    const auto selectedTime = app->state->getPreparationTime();
+    for (unsigned int i = 0; i < Constants::Params::preparationTimes.size(); ++i) {
+        if (selectedTime == Constants::Params::preparationTimes[i]) {
```

Comment:
Besides, thsi could probably also be an `std::find()` usage

Reply:
done